tests/test_filesystem.py

```diff
@@ -45,7 +45,7 @@ def test_compression_filesystems(compression_fs_class, gz_file, bz2_file, lz4_fi
     assert isinstance(fs, compression_fs_class)
     expected_filename = os.path.basename(input_path)
     expected_filename = expected_filename[: expected_filename.rindex(".")]
-    assert fs.ls("/") == [expected_filename]
+    assert fs.ls("/", detail=False) == [expected_filename]
```

Comment:
Since we never use `fs.ls` in the datasets library (but `fs.glob` for ZIP archive), what about aligning the test with what we really use from `fsspec`? This way we will avoid unnecessarily breaking our CI with any other future breaking change on `fs.ls`.
```suggestion
    assert fs.glob("*") == [expected_filename]
```